db/migrate/20230117091239_add_account_closed_fields_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddAccountClosedFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users, bulk: true do |t|
+      t.datetime :account_closed_at
```

Comment:
If you can be bothered - I have seen benefits from grouping db fields with a common prefix related to the feature. So having it be `closed_at` follows that pattern. Another reason is `Account` is a business term seen in the browser but we don't have data model for it, so this avoids an ambiguity.